compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/viewinterop/SwingInteropContainer.desktop.kt

```diff
@@ -144,40 +144,37 @@ internal class SwingInteropContainer(
 
     override fun place(holder: InteropViewHolder) {
         val group = holder.group
-
         if (interopComponents.isEmpty()) {
             snapshotObserver.start()
         }
 
-        // [ComposeSceneMediator] might keep extra components in the same container.
-        // Reed it before modifying both [interopComponents] and [root] to get consistent result
-        val nonInteropComponents = root.componentCount - interopComponents.size
+        // Note that compose state must be read here, but AWT/Swing state must be read inside
+        // scheduleUpdate
 
         // Add this component to [interopComponents] to track count and clip rects
-        val alreadyAdded = group in interopComponents
-        if (!alreadyAdded) {
-            interopComponents[group] = holder
-        }
+        val isNewView = interopComponents.putIfAbsent(group, holder) == null
```

Comment:
let's avoid just "view" in naming here, it's too vague in this context
```suggestion
        val isNewInteropView = interopComponents.putIfAbsent(group, holder) == null
```

Reply:
Done